src/components/common/Footer/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC } from 'react';
+import { ReactComponent as LogoCarbonDeFi } from 'assets/logos/carbondefi.svg';
+
+export const Footer: FC = () => {
+  return (
+    <footer className="bg-body fixed bottom-0 left-0 z-10 hidden h-65 w-full items-center justify-start gap-8 border-t border-darkGrey py-16 px-20 md:flex">
```

Comment:
I would avoid using fixed positions when possible.
If you don't use fixed, you wont' need the w-full anymore.
I would avoid setting a defined height but use only paddings instead to make it future proof if we want to add more content.
I think the color used for the border is emphasis (#212123 in figma)